services/gateway/apis/http-version3/swagger/definitions/newsfeed.json

```diff
@@ -0,0 +1,94 @@
+{
+    "news": {
+        "type": "object",
+        "required": [
+            "author",
+            "content",
+            "source",
+            "sourceId",
+            "createdAt",
+            "modifiedAt",
+            "title",
+            "url",
+            "image_url"
+        ],
+        "properties": {
+            "author": {
+                "type": "string",
+                "example": "LiskHQ",
+                "description": "Blog post author."
+            },
+            "content_t": {
+                "type": "string",
+                "example": "RT @newsbtc: Lisk.js 2021 Recap https://t.co/QpZOkBfrgA",
+                "description": "Blog post text."
+            },
+            "source": {
+                "type": "string",
+                "example": "twitter_lisk",
+                "description": "Source name"
+            },
+            "source_id": {
+                "type": "string",
+                "example": "4584a7d2db15920e130eeaf1014f87c99b5af329",
+                "description": "External ID of the blog post."
+            },
+            "createdAt": {
+                "type": "integer",
+                "example": 1623053809,
+                "description": "Creation time."
+            },
+            "modifiedAt": {
+                "type": "integer",
+                "format": "hex",
+                "example": 1623053809,
+                "description": "Modification time."
+            },
+            "title": {
+                "type": "string",
+                "example": "Financial Update for January 2021",
+                "description": "Blog post title."
+            },
+            "url": {
+                "type": "string",
+                "format": "id",
+                "example": "https://t.co/QpZOkBfrgA",
+                "description": "The id of the previous block of the chain."
+            },
+            "image_url": {
+                "type": "string",
+                "example": "https://t.co/QpZOkBfrgA.jpg",
+                "description": "Original URL with the blog post."
+            }
+        }
+    },
+    "NewsfeedWithEnvelope": {
+        "type": "object",
+        "required": [
+            "data",
+            "meta"
+        ],
+        "properties": {
+            "data": {
+                "description": "List of news items",
+                "type": "array",
+                "items": {
+                    "$ref": "#/definitions/news"
+                }
+            },
+            "meta": {
+                "type": "object",
+                "required": [
+                    "count"
+                ],
+                "properties": {
+                    "count": {
```

Comment:
Could we please extend the `meta` definition to include `offset` and `total` as well?
Possibly use `"$ref": "#/definitions/Pagination"`??